sycl/include/sycl/ext/intel/experimental/fpga_lsu.hpp

```diff
@@ -101,32 +85,16 @@ template <class... _mem_access_params> class lsu final {
     check_space<_space>();
     check_store();
 #if defined(__SYCL_DEVICE_ONLY__) && __has_builtin(__builtin_intel_fpga_mem)
-    // Get latency control properties
-    using _latency_anchor_id_prop = typename detail::GetOrDefaultValT<
-        _propertiesT, latency_anchor_id_key,
-        detail::defaultLatencyAnchorIdProperty>::type;
-    using _latency_constraint_prop = typename detail::GetOrDefaultValT<
-        _propertiesT, latency_constraint_key,
-        detail::defaultLatencyConstraintProperty>::type;
-
-    // Get latency control property values
-    static constexpr int32_t _anchor_id = _latency_anchor_id_prop::value;
-    static constexpr int32_t _target_anchor = _latency_constraint_prop::target;
-    static constexpr latency_control_type _control_type =
-        _latency_constraint_prop::type;
-    static constexpr int32_t _relative_cycle = _latency_constraint_prop::cycle;
-
-    int32_t _control_type_code = 0; // latency_control_type::none is default
-    if constexpr (_control_type == latency_control_type::exact) {
-      _control_type_code = 1;
-    } else if constexpr (_control_type == latency_control_type::max) {
-      _control_type_code = 2;
-    } else if constexpr (_control_type == latency_control_type::min) {
-      _control_type_code = 3;
+    _T *P = Ptr;
+    if constexpr (!std::is_same_v<
+                      _propertiesT,
+                      oneapi::experimental::detail::empty_properties_t>) {
```

Comment:
While we are at it, we might as well make use the new extension alias.

```suggestion
                      oneapi::experimental::empty_properties_t>) {
```

Reply:
Addressed